ui/app/templates/components/tool-rewrap.hbs

```diff
@@ -6,35 +6,35 @@
   </p.levelLeft>
 </PageHeader>
 
-{{#if rewrap_token}}
+{{#if @rewrap_token}}
   <div class="box is-sideless is-fullwidth is-marginless">
     <div class="field">
       <label for="wrap-info" class="is-label">Rewrapped token</label>
       <div class="control">
-        <Textarea @value={{rewrap_token}} @readonly={{true}} class="textarea" @id="wrap-info" @name="wrap-info" data-test-tools-input="rewrapped-token" />
+        <Textarea @value={{@rewrap_token}} @readonly={{true}} class="textarea" @id="wrap-info" @name="wrap-info" data-test-tools-input="rewrapped-token" />
       </div>
     </div>
   </div>
   <div class="field is-grouped box is-fullwidth is-bottomless">
     <div class="control">
-      <CopyButton @clipboardText={{rewrap_token}} @class="button is-primary" @buttonType="button" @success={{action (set-flash-message "Token copied!")}}>
+      <CopyButton @clipboardText={{@rewrap_token}} @class="button is-primary" @buttonType="button" @success={{action (set-flash-message "Token copied!")}}>
         Copy
       </CopyButton>
     </div>
     <div class="control">
-      <button {{action 'onClear'}} type="button" class="button">
+      <button {{on "click" (action this.onClear)}} type="button" class="button">
```

Comment:
we *should* be able to just do `{{on "click" this.onClear}}` but if that doesn't work for some reason I'd be happy to pair about it